spec/ace/config_spec.rb

```diff
@@ -0,0 +1,176 @@
+# frozen_string_literal: true
```

Comment:
These tests are part of #17 so I do not think they are required as part of this PR as it is not related to remote tasks, or close #17 and move this to spec/unit/ace folder

Reply:
This work is part of the completion of remote tasks. I did miss that https://github.com/puppetlabs/ace/pull/17 was up, but as it is a direct port of the Bolt test it doesn't give us much. The changes made here provide us with the additional testing required. I would suggest closing https://github.com/puppetlabs/ace/pull/17

This continues to be WIP, I've updated the title as such.